src/Http/Controllers/Api/FormController.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tipoff\Forms\Http\Controllers\Api;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Validation\Rule;
+use Tipoff\Forms\Models\Contact;
+use Tipoff\Support\Http\Controllers\Api\BaseApiController;
+
+class FormController extends BaseApiController
+{
+    public function __invoke(Request $request)
+    {
+        $domain = request('__amp_source_origin');
+
+        $basic = [
+            'contact',
+            'employment',
+        ];
+        if (in_array($request->form_type, $basic)) {
+            $validator = Validator::make($request->all(), [
+                'form_type' => [    'required',
+                    Rule::in([
+                        'contact',
+                        'employment',
+                    ]),
+                ],
+                'first_name' => 'required',
+                'last_name' => 'required',
+                'email_address' => 'required|email',
+                'phone' => 'required',
+                'message' => 'nullable|max:3000',
+            ]);
+        } else {
+            $validator = Validator::make($request->all(), [
+                'form_type' => [    'required',
+                    Rule::in([
+                        'on-the-run',
+                        'parties',
+                        'reservations',
+                        'team-building',
+                    ]),
+                ],
+                'first_name' => 'required',
+                'last_name' => 'required',
+                'email' => 'required|email',
+                'phone' => 'required',
+                'message' => 'nullable|max:3000',
+                'company_name' => 'nullable|string',
```

Comment:
The original project contained more fields, but as discussed, they would live as custom fields. The default validator should only account for what is required regardless.